src/menubar.ts

```diff
@@ -360,6 +409,11 @@ export default class UI extends EventEmitter {
         ]
       },
       { type: 'separator' },
+      {
```

Comment:
I have no strong feelings here, but I envisioned this between version number and email. :) What do you think?